pandas/core/strings.py

```diff
@@ -2370,6 +2370,16 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
     0          this is a regular sentence        None
     1  https://docs.python.org/3/tutorial  index.html
     2                                 NaN         NaN
+
+    Remember to escape special characters when explicitly using regular
+    expressions.
+
+    >>> s = pd.Series(["1+1=2", np.nan])
```

Comment:
Can you get rid of `np.nan` in this example? I think it just distracts from what is going on

Reply:
I agree with this comment.  Let's remove, and this should be good to go.

Reply:
Done. Kindly check the changes once more. Thanks!